opennebula/shared_schemas.go

```diff
@@ -439,6 +425,9 @@ func flattenNIC(nic shared.NIC) map[string]interface{} {
 func flattenDisk(disk shared.Disk) map[string]interface{} {
 
 	size, _ := disk.GetI(shared.Size)
+	if size == -1 {
```

Comment:
how is it possible to have `-1` as disk size ?

Reply:
It's related to Goca behavior, when the attribute attribute is not found, `GetI` returns `-1`
The goal is to avoid reading `-1` as a disk size in case of a ONE template with a disk defined without `SIZE` attribute.
In this case, at next plan terraform show a diff `-1` -> `0`

Reply:
ok got it thanks